crosstool/setup.bzl

```diff
@@ -27,7 +27,7 @@ def _apple_cc_autoconf_toolchains_impl(repository_ctx):
     else:
         repository_ctx.file("BUILD", "# Apple CC toolchain autoconfiguration was disabled because you're not running on macOS")
 
-_apple_cc_autoconf_toolchains = repository_rule(
+apple_cc_autoconf_toolchains = repository_rule(
```

Comment:
If it's a concern that these symbols are now visible, I could restructure the file further and adopt `visibility()` to prevent that.

Reply:
I would prefer not to increase visibility on any of our symbols.

Reply:
I rearranged the contents so that I can keep the symbols private. This is assuming that your minimum supported version of Bazel is 6 (for load visibility).

Reply:
Currently our minimum is Bazel 5.

Reply:
I see, then I don't see a way to enforce visibility. Is the name `setup_internal.bzl` good enough or should I move the file to some `internal` or `private` subdirectory?

Reply:
Under an `internal/` directory will add some buildifier enforcement as well, so that should be good enough.

Reply:
I moved it.